roles/ceph-osd/tasks/start_osds.yml

```diff
@@ -32,5 +34,6 @@
   service:
     name: ceph-osd@{{ item }}
     state: started
+    enabled: true
```

Comment:
👎 let's check with @alfredodeza but we removed that specifically because this is being done by ceph-disk already and we shouldn't change the behavior.

Reply:
Understood.